pkg/backup/item_backupper.go

```diff
@@ -206,7 +206,10 @@ func (ib *defaultItemBackupper) backupItem(logger logrus.FieldLogger, obj runtim
 					return err
 				}
 
-				ib.additionalItemBackupper.backupItem(log, additionalItem, gvr.GroupResource())
+				err = ib.additionalItemBackupper.backupItem(log, additionalItem, gvr.GroupResource())
```

Comment:
When we are only checking the returned error and aren't doing anything else with it, we usually use the syntax `if err := foo(); err != nil {`

Reply:
Right. I'll do the same for the three if:s above at the same time, then?

Reply:
Ah, sorry. That doesn't work though, since they set other vars. Never mind, then, just fixing this one :)